src/output/details.rs

```diff
@@ -362,7 +361,7 @@ impl<'a> Render<'a> {
     }
 
     fn render_xattr(&self, xattr: &Attribute, tree: TreeParams) -> Row {
-        let name = TextCell::paint(self.theme.ui.perms.attribute, format!("{} (len {})", xattr.name, xattr.size));
+        let name = TextCell::paint(self.theme.ui.perms.attribute, format!("{} : {}", xattr.name, xattr.value));
```

Comment:
I believe there’s no reason to put a space before the xattr name here.

```suggestion
        let name = TextCell::paint(self.theme.ui.perms.attribute, format!("{}: {}", xattr.name, xattr.value));
```

Reply:
I'd like to keep a space between the name and the colon, otherwise it can be ambiguous whether the colon is a separator from exa or belongs to the extended attribute name.

Reply:
I think I’d prefer if it used the same syntax as other tools. [`getfattr`](https://man.archlinux.org/man/getfattr.1) uses `name="value"`, and I believe we should use the same convention, except if there is a good reason to do otherwise.

Reply:
switched to `$name="$value"`